src/styles/comp.scss

```diff
@@ -0,0 +1,73 @@
+.comp {
+  display: flex;
+  flex-direction: column;
+  margin-inline: $margin-inline;
+  font-family: $font-family;
+
+  &__name {
+    padding-top: 207px;
+    font-weight: 500;
+    font-size: 12px;
+    align-self: center;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    background-position: 0 33px;
+    color: $text-color-strong;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    margin-top: 4px;
+    color: $text-color;
+  }
+
+  &__rate {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &--stars {
+      display: flex;
+
+      &__star {
+        width: 16px;
+        height: 16px;
+        background-image: url(../images/star.svg);
+        background-repeat: no-repeat;
+        background-position: center;
+        margin-right: 4px;
+      }
```

Comment:
The class `comp__rate--stars__star` is not following BEM conventions correctly. The double underscore is used for elements, and it should be `comp__rate__star` if it's an element of `comp__rate`, or `comp__rate--star` if it's a modifier.